RealmJS.podspec

```diff
@@ -0,0 +1,97 @@
+end
+
+Pod::Spec.new do |s|
+  s.name                   = "RealmJS"
+  s.version                = package['version']
+  s.summary                = package['description']
+  s.license                = package['license']
+
+  s.authors                = package['author']
+  s.homepage               = package['homepage']
+  s.platform               = :ios, '9.0'
+
+  s.source                 = { :git => git_path, :tag => "v#{s.version}", :submodules => true }
+
+  s.prepare_command        = <<-CMD
+    # Run `npm ci` to install the dependencies the download script needs, if not installed as a node_module
+    ([ -d '../../node_modules' ] || npm ci) && node ./scripts/download-realm.js ios --sync;
+    # Clean up from any previous builds
+    rm -rf include
+    # Build up the include directory
+    mkdir -p include include/jsc include/impl/apple include/util/apple include/sync/impl/apple
+    # Copy headers from Realm JS
+    cp src/*.hpp include
+    # Copy headers from RealmReact
+    cp react-native/ios/RealmReact/*.h include
+    # Copy headers from the vendor root directory
+    cp vendor/*.hpp include
+    # Copy headers from core + sync
+    cp -r vendor/realm-ios/include include/core+sync
+    # Copy headers from object store
+    cp src/object-store/src/*.hpp include
+    cp src/object-store/src/sync/*.hpp include/sync
+    cp src/object-store/src/sync/impl/*.hpp include/sync/impl
+    cp src/object-store/src/sync/impl/apple/*.hpp include/sync/impl/apple
+    cp src/object-store/src/impl/*.hpp include/impl
+    cp src/object-store/src/impl/apple/*.hpp include/impl/apple
+    cp src/object-store/src/util/*.hpp include/util
+    cp src/object-store/src/util/apple/*.hpp include/util/apple
+    cp src/object-store/external/json/*.hpp include
+  CMD
+
+  s.source_files           = 'src/*.cpp',
+                             'src/jsc/*.{cpp,hpp,h}',
+                             'src/ios/*.mm',
+                             'src/object-store/src/*.cpp',
+                             'src/object-store/src/sync/*.cpp',
+                             'src/object-store/src/sync/impl/*.cpp',
+                             'src/object-store/src/sync/impl/apple/*.cpp',
+                             'src/object-store/src/impl/*.cpp',
+                             'src/object-store/src/impl/apple/*.cpp',
+                             'src/object-store/src/util/*.cpp',
+                             'src/object-store/src/util/apple/*.cpp',
+                             'react-native/ios/RealmReact/*.mm',
+                             'vendor/*.cpp'
+  
+  s.frameworks             = 'Security', 'JavaScriptCore'
+  s.library                = 'c++', 'z'
+  s.compiler_flags         = '-DREALM_HAVE_CONFIG -DREALM_ENABLE_SYNC'
+  s.header_mappings_dir    = 'include'
+  s.preserve_paths         = 'include', 'scripts'
+  s.pod_target_xcconfig    = { 'CC' => '$(PODS_TARGET_SRCROOT)/scripts/ccache-clang.sh',
```

Comment:
I don't think we should require users to install ccache.

Reply:
It doesn't the https://github.com/realm/realm-js/blob/master/scripts/ccache-clang.sh bash scripts checks for the existence of ccache and falls back to clang (or clang++).